composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require": {
     "php": ">=5.3.3",
-    "apache/log4php": "2.3.0"
+    "apache/log4php": "2.3.0",
```

Comment:
@johanwilfer Yes, it was to maintain backwards compatibility. However, that may be fixed in the 1.80.0 branch as Marcelo mentioned when closing this PR.

Thanks for the review though!